habitat-aspnet-sample/plan.sh

```diff
@@ -0,0 +1,49 @@
+pkg_name=habitat-aspnet-sample
+pkg_origin=core
+pkg_version=0.1.9
+pkg_source=https://github.com/mwrock/${pkg_name}/archive/v${pkg_version}.tar.gz
+pkg_shasum=50ffcbe1c0b49acb84365c2157445047b360345c7361e67827093f962852de4d
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
+pkg_description="A sample asp.net core app for linux"
+pkg_bin_dirs=(bin)
+pkg_expose=(8090)
+
+pkg_deps=(
+  core/dotnet-core
+  core/gcc
+  core/glibc
+  core/gcc-libs
+  core/icu/52.1
```

Comment:
Are all these deps needed so we can `patchelf`? Wouldn't just including `core/dotnet-core` include these transitively or do we need to include them here as well?